site/content/docs/developer/building-zap-with-intellij-idea.md

```diff
@@ -79,6 +79,51 @@ Note that the add-ons in _zap-extensions_ all have their own set of tasks in add
 
 ## Tips and Tricks
 
+### Connecting to a session database
+
+You can use the Database tab in IntelliJ IDEA to connect to a persisted ZAP session.
+By default, ZAP uses [HSQLDB](https://hsqldb.org/) to store its session data.
+There is also experimental support for MySQL and PostgreSQL but we won't be discussing that here.
+
+Since ZAP locks its session file, it is only possible to view or modify the database when ZAP doesn't have the session open.
+By default, ZAP stores its session files under `session/` in its [default directory](/faq/what-is-the-default-directory-that-zap-uses/).
+
+To open a session database in IDEA, click on the Database tab, and add a new HSQLDB data source.
```

Comment:
IMHO we should just document the essential on the ZAP side and for the remaining link to official documentation (e.g. https://www.jetbrains.com/help/idea/relational-databases.html ), less maintenance in the long term (i.e. we have DB wiki pages that are not touched since 2015).

Reply:
Agreed, I looked at those as reference for these updates as well. Is it worth moving them to a new dev docs page from the wiki and linking to that as well as the intellij docs?

Reply:
That sounds good, some of the info will be common to all IDEs.